keyboards/id67/keymaps/via/config.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2021 Tybera
+/* Copyright 2021 thewerther
```

Comment:
```suggestion
/* Copyright 2021 Tybera
 * Copyright 2022 thewerther
```

It is probably good to retain name(s) of previous contributors in the source.

Reply:
This diff is a side effect of moving the files in and out of a folder. But, the advice is still valid